.pipelines/cni/lsg/pipeline.yaml

```diff
@@ -0,0 +1,100 @@
+parameters:
+- name: KernelVersion
+  type: string
+  default: 1000
+- name: KernelType
+  type: string
+  default: linux-azure-edge
+- name: ProposedRepoVersion
+  type: string
+  default: proposed
+
+pr: none
+trigger: none
+
+# Placeholder for when we want to enable automatic queuing of pipeline from LSG's pipeline.
+# Fully functional to queue off of ACN PR Pipeline
+# resources:
+#   pipelines:
+#   - pipeline: ACN-PR # Name of the pipeline resource.
+#     source: \Custom\Networking\ContainerNetworking\Azure Container Networking PR # The name of the pipeline referenced by this pipeline resource.
+#     trigger: true # Run CNI | LSG Integration pipeline when any run of LSG defined pipeline completes
+
+
+stages:
+  - stage: setup
+    displayName: Setup
+    jobs:
+      - job: env
+        displayName: Setup
+        pool:
+          name: "$(BUILD_POOL_NAME_DEFAULT)"
+        steps:
+          - script: |
+              echo "Setting up environment"
+              go version
+              echo "##vso[task.setvariable variable=commitID;isOutput=true]$(echo $(make revision)-$(date "+%d%H%M"))"
+              echo "##vso[task.setvariable variable=cnsVersion;isOutput=true]$(CNS_VERSION)"
+              echo "##vso[task.setvariable variable=cniVersion;isOutput=true]$(CNI_VERSION)"
+
+              echo "------"
+              echo Queued by $(Build.QueuedBy)
+              echo "Runtime Parameters passed"
+              echo "KernelType : ${{ parameters.KernelType }}"
+              echo "KernelVersion : ${{ parameters.KernelVersion }}"
+              echo "ProposedRepoVersion : ${{ parameters.ProposedRepoVersion }}"
+              echo "------"
+              echo "KernelType Check"
+              KernelType=${{ parameters.KernelType }}
+              if [ $KernelType = 'proposed-azure' ]; then
+                echo "KernelType is $KernelType , change to linux-azure"
```

Comment:
Does this mean we should exit the pipeline ?

Reply:
No, it is just remapping the what the incoming value is. LSG's has dependency on updating these kernel type with that given key, which updates that kernel value on the DB. All other kernelTypes are 1:1 with the same key and value, so we do not have this problem.